cmake/FindMAVLink.cmake

```diff
@@ -0,0 +1,43 @@
+# - Try to find  MAVLink
+# Once done, this will define
+#
+#  MAVLINK_FOUND        : library found
```

Comment:
I'm not a cmake expert, but `MAVLINK_FOUND` is not being set in this file. This needs to be set depending on if the search was successful or not, right?

Reply:
https://cmake.org/cmake/help/v3.0/command/find_package.html. if the FindMAVLink fails, it returns NOT_FOUND, else ir returns FOUND.

Reply:
You don't have to explicitly set it on the `.cmake` config file.